kie-server-parent/kie-server-api/src/test/java/org/kie/server/api/marshalling/json/JSONMarshallerExtensionTest.java

```diff
@@ -55,4 +58,25 @@ public void testCustomExtensionMarshaller() {
         assertEquals(rd, request);
         System.out.println(rd);
     }
+
+    @Test
+    public void testObjectInsideCommand() {
+        Set<Class<?>> extraClasses = new HashSet<Class<?>>();
+        extraClasses.add(CustomPerson.class);
+        Marshaller marshaller = MarshallerFactory.getMarshaller(extraClasses, MarshallingFormat.JSON, this.getClass().getClassLoader());
+
+        CustomPerson john = new CustomPerson("John", 20);
+        InsertObjectCommand command = new InsertObjectCommand(john);
+        String marshall = marshaller.marshall(command);
+
+        // verify if it's processed by JSONMarshallerExtensionCustomPerson serializer
+        assertTrue("marshalled string should contain a customized value", marshall.contains("John is CustomPerson"));
```

Comment:
What about replacing this assertion with something like `Assertions.assertThat(marshall).contains("John is CustomPerson");` ?
That way it will be directly visible what is the actual content of marshall variable.